src/commands/deploy.js

```diff
@@ -112,6 +112,11 @@ DeployCommand.flags = {
     char: 'f',
     description: 'Skip deploy confirmation.',
   }),
+  projectId: flags.string({
+    char: 'p',
+    description: 'Forest project ID.',
```

Comment:
Should we validate the description with the product team?

In `init` command, the text is:

> The id of the project you want to init.

`push` command:

> The id of the project to work on.


So it could be: 

```suggestion
    description: 'The id of the project you want to deploy.',
```

(My suggestion is not that good I guess. Maybe just: `The id of the project to work on`)

Anyway `create` and environments commands use "Forest project ID"...

As you wish then!

Reply:
I don't think this should be taken into a whole process of validation. But indeed your suggestion is way better than what I did 👍